openapi/openapi.yaml

```diff
@@ -0,0 +1,62 @@
+openapi: 3.1.0
+info:
+  version: 1.0.0
+  title: Deep Learning Playground API
+  termsOfService: https://example.com/terms/
+  contact:
+    email: contact@example.com
+    url: http://example.com/contact
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  x-logo:
+    url: https://redocly.github.io/openapi-template/logo.png
+  description: >
+    This is an **example** API to demonstrate features of the OpenAPI
+    specification.
+
+tags:
+  - name: dataset
+    description: Everything about dataset
+  - name: s3
+    description: Everything about s3
+  - name: test
+    description: Test API endpoints
+  - name: trainspace
+    description: Everything about trainspace
+servers:
+  - url: https://example.com/api/v1
+paths:
+  /dataset/defaultDataset:
+    $ref: paths/dataset/defaultDataset.yaml
```

Comment:
do tags get linked to yaml files? @farisdurrani

Reply:
Yup, we'd have to put the tag manually inside each yaml file